style.css

```diff
@@ -0,0 +1,1057 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+* {
+    font-family: 'Montserrat';
+    margin-left: 0;
+    margin-right: 0;
+}
+
+:root {
+    --lilac: #BCB6FF;
+    --pink: #E55381;
+    --gray: #9ca09f;
+    --gray-fade: #e4e7e6;
+    --orange: #FE5F55;
+    --dark-purple: #3D2645;
+    --dark-pink: #DA4167;
+    --off-white: #F0EFF4;
+    --royal-blue: #0A2463;
+    --blue: #235fe0;
+    --sky-blue: #3E92CC;
+    --rich-black: #070600;
+    --magenta: #CA1551;
+    --success: #5cb85c;
+    --green: #158645;
+    --danger: #d9534f;
+    --info: #5bc0de;
+    --warning: #f0ad4e;
+    --success-fade: #5cb85c27;
+    --danger-fade: #eb413b27;
+    --info-fade: #5bc0de18;
+    --warning-fade: #f0ac4e1e;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    position: fixed;
+    z-index: 1;
+    background-color: white;
+    width: 100%;
+    top: -0.1em;
+    box-shadow: 10px 2px 10px var(--gray-fade);
+}
```

Comment:
though this works perfectly, but it'd be great if we don't touch HTML elements directly